src/js/components/Notification/propTypes.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    title: PropTypes.string,
+    message: PropTypes.string,
+    status: PropTypes.string,
```

Comment:
We are limiting `status` to a limited set of accepted values, correct? If so, this PropType definition should be narrowed and aligned with the Typescript definition.
```suggestion
    status: PropTypes.oneOf(['critical', 'warning', 'normal', 'unknown']),
```